torchinductor/graph.py

```diff
@@ -208,6 +208,14 @@ def call_function(self, target, args, kwargs):
         if target is operator.getitem and isinstance(args[0], (list, tuple)):
             return super().call_function(target, args, kwargs)
 
+        print(target)
+        if target == torch.ops.math.size:
```

Comment:
I think we could just put these in `lowerings.py` and use the normal lowerings dict.